acasclient/experiment.py

```diff
@@ -86,14 +112,27 @@ def protocol_name(self) -> str:
                 label_kind="protocol name")
         return protocol_name["labelText"]
 
+    @property
+    def column_information(self) -> list:
+        data = get_entity_values_by_state_type_kind_value_type(
+                entity=self,
+                state_type="metadata",
+                state_kind="data column order",
+                value_type="codeValue")
+        return data
+
     @property
     def result_kinds(self) -> list:
         data = get_entity_values_by_state_type_kind_value_type(
```

Comment:
Can we remove this call now that you're getting the data from `self.column_information` rather than `data`?

Reply:
Yes, thanks, missed removing that. Fixed.